adding option to provide operator private key as keystore file encryp… (#1019)
bloxapp/ssv: docs/DEV_GUIDE.md

```

#### Generating an Operator Key
To generate an operator key, you can use the `ssvnode generate-operator-keys command`. This command can generate the key in two ways: raw format or encrypted format.

**IMPORTANT**: The raw format is **NOT recommended** for production use, as it can expose sensitive data. Use the encrypted format for added security.

**Raw Operator Key Generation:**
This is the default mode of operation and does not require any additional parameters.
```bash
$ ./bin/ssvnode generate-operator-keys
```
This will generate an operator key in raw format.

**Encrypted Operator Key Generation:**

> To generate an operator key in encrypted format, use the `--encrypted-password option` followed by your desired password.

Reviewer: you used `encryption-password` in the command. this doc won't work, anyway suggested to change to something much shorter. 
```suggestion
To generate an operator key in encrypted format, use the `--password ****` followed by your desired password.
```

```bash

> $ ./bin/ssvnode generate-operator-keys --encrypted-password=<your-password-here>

Reviewer: same as above, you used `encrypted` instead the correct command `encryption` anyway I suggest `password` only.
```suggestion
$ ./bin/ssvnode generate-operator-keys --password=<your-password-here>
```

```

#### Generating an Operator Key

> To generate an operator key, you can use `ssvnode generate-operator-keys`. This command can generate the key in two ways: raw format or encrypted format.

Reviewer 2: ```suggestion
To generate an operator key, you can use `./bin/ssvnode generate-operator-keys`. This command can generate the key in two ways: raw format or encrypted format.
```

**IMPORTANT**: The raw format is **NOT recommended** for production use, as it can expose sensitive data. Use the encrypted format for added security.

**Raw Operator Key Generation:**
This is the default mode of operation and does not require any additional parameters.
```bash
$ ./bin/ssvnode generate-operator-keys
```
This will generate an operator key in raw format.

**Encrypted Operator Key Generation:**

> To generate an operator key in encrypted format, use the `--password-file option` followed by your desired password.

Reviewer 2: ```suggestion
To generate an operator key in encrypted format, use the `--password-file` option followed by your password file.
```

```bash

> $ ./bin/ssvnode generate-operator-keys --password-file=<your-password-here>

Reviewer 2: ```suggestion
$ ./bin/ssvnode generate-operator-keys --password-file=path/to/your/file
```

```

> Please replace `<your-password-here>` with your chosen password. This will generate an operator key in encrypted format.

Reviewer 2: ```suggestion
Please replace `path/to/your/file` with your password file. This will generate an operator key in encrypted format.
```

    ```
   This private key can be generated using the CLI command ./bin/ssvnode generate-operator-keys which generates the RSA public key and private key in base64 format.
   However, for a more secure approach, it's recommended to use an encrypted PEM file with a password. This adds an extra layer of security as the private key is not only encoded but also encrypted.

2) As a Keystore PEM file encrypted with a password:
    ```yaml
    KeyStore:
     PrivateKeyFile: /path/to/your/file
     PasswordFile: /path/to/your/file
    ```
   You can convert the base64-encoded private key to an encrypted PEM file using the following OpenSSL command:
    ```yaml
   Set OpenSSL version to LibreSSL first. This is the only pem format that can be decrypted.
   

>     openssl rsa -aes256 -in <path-to-base64-private-key-file> -out <path-to-encrypted-pem-file> -passout pass:<password>

Reviewer: update this please

    ```
   This private key can be generated using the CLI command `./bin/ssvnode generate-operator-keys`. However,
   this method merely encodes the private key, it doesn't provide additional encryption.
   For a more secure approach, it's recommended to use an encrypted keystore file with a password.

2) As an Encrypted Keystore file:
    ```yaml
    KeyStore:
     PrivateKeyFile: /path/to/your/file
     PasswordFile: /path/to/your/file
    ```
   In this approach, the private key is not only encoded but also encrypted, adding an extra layer of security.

>    To generate the encrypted keystore file, run the `./bin/ssvnode generate-operator-keys` command with the `--password-file=****` flag.

Reviewer 2: ```suggestion
   To generate the encrypted keystore file, run the `./bin/ssvnode generate-operator-keys` command with the `--password-file=path/to/your/file` flag.
```

>    Replace <path/to/your/file> with your desired output file path for the encrypted keystore and the corresponding password file.

Reviewer 2: ```suggestion
   Replace `path/to/your/file` with password file path.
```
